### ISSUE TITLE
Edit/Create links for docs pages should point to authoritative sources

### ISSUE THREAD

**Issue by austinlparker** [open-telemetry/opentelemetry.io#542]
Currently, on each doc page there's a set of links in the upper-right above the doc ToC for opening issues/creating child pages for a docs page.
![image](https://user-images.githubusercontent.com/4140740/114781208-643e2d00-9d46-11eb-8f61-018c505848e2.png)

It would be nice if these could point to the upstream source (i.e., `open-telemetry/opentelemetry-java/website_docs`) rather than the website repo itself. 

This would require some Docsy hacking, as these are set in `themes/docsy/layouts/partials/page-meta-links.html`.

**Comment by harshita19244** [comment]
@austinlparker I would like to work on it, could you assign it to me?

**Comment by harshita19244** [comment]
I had a question, to make these changes, do I need to directly modify the code in the Docsy repo? Since the `themes/docsy `points to the actual Docsy repo itself. Do I send a pr there? @austinlparker

**Comment by austinlparker** [comment]
I don't think we should change the underlying theme (as that means we wouldn't be able to update it easily) - Hugo has a lookup order for partials, so we could customize it by creating a copy of it then tweaking it. See https://gohugo.io/templates/partials/ for more information on how the template lookups work.

**Comment by chalin** [comment]
Hi all. Need a hand with this? I've implemented this in other website repos, and would be glad to set this up here too (once I'm done with basic infrastructure cleanup).

**Comment by chalin** [comment]
Mind if I pickup work on this issue @harshita19244?

**Comment by chalin** [comment]
@austinlparker - By the way, there's a "feature interaction" with multi-language support -- which the website is currently ready for, though currently on `en` is supported.  That is, language repos should probably be setup for multi-language support as well. Thoughts?

**Comment by chalin** [comment]
Self-assigning. The first couple of PRs are landing soon.

**Comment by chalin** [comment]
> Mind if I pickup work on this issue @harshita19244?

@harshita19244 - FYI, I'll be completing the resolution of this issue. Thanks for your original interest!

**Comment by chalin** [comment]
See #730
